src/components/Header.jsx

```diff
@@ -17,8 +17,9 @@ export default function Header() {
 
   return (
     <header className="min flex h-14 w-full items-center justify-between bg-secondary-1 px-4 py-4 md:h-[4.25rem] md:px-20">
-      <Logo withText class="h-full cursor-pointer" />
-
+      {window.innerWidth>767 && (<Logo withText className="h-full cursor-pointer" />)}
+      {window.innerWidth<768 && (<Logo className="h-full cursor-pointer" />)}
```

Comment:
Pls more attention with this few stroke, couse i dont know how i can do it better with hiding logo text if u know i will pleasure to remake

Reply:
We can use this:
https://www.npmjs.com/package/react-responsive

but let's just use without the logo. it's ok. Let's keep it simple :)